src/azure-cli-core/azure/cli/core/auth/util.py

```diff
@@ -53,12 +53,21 @@ def aad_error_handler(error, **kwargs):
     raise AuthenticationError(error_description, msal_error=error, recommendation=recommendation)
 
 
-def _generate_login_command(scopes=None, claims_challenge=None):
+def _generate_login_command(tenant=None, scopes=None, claims_challenge=None):
```

Comment:
Shall we append the tenant info in re-login recommendation msg only for some error code (for example AADSTS50076)?

Now it seems all failure cases will append this except for 7000215
https://github.com/Azure/azure-cli/blob/cf5633149785a352cbf9788f1f2e0820e7612552/src/azure-cli-core/azure/cli/core/auth/util.py#L43-L47

Reply:
My concern is whether there will be any cases that adding tenant id during login is misleading? If no such case, then I'm good with this

Reply:
I cannot think of a case where adding tenant ID is misleading.